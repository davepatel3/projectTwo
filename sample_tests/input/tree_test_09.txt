string
allocate
S~>7X_7.ySaZ
0
allocate
i@PS`$)O(hW
1
allocate
n~4+AG>
1
remove
5
allocate
c_:Z
1
set_parent
3
2
remove
2
ref_node
4
allocate
jLr[v,V
1
allocate
S2UU(B=_swVODY2s
3
allocate
2I?^_z>
3
allocate
/Wx6L
4
allocate
B0**wW_4]`_%5
2
ref_node
6
remove
1
allocate
;iJ</AN}M,
0
allocate
[GjxM
6
set_parent
4
0
ref_node
6
ref_node
6
allocate
R/['hYth
6
allocate
;q)_X**NW3fr
0
set_parent
6
2
allocate
HsG${gc(uQpLQ?
2
set_parent
9
69618407330165
remove
2
remove
4
peek_nodes
allocate
:L8U]"]5V)ja*ms
0
allocate
h*'=
0
remove
3
allocate
DbFx`H,8P\
1
set_parent
7
0
remove
1
remove
7
allocate
P#{`BKV
0
allocate
f/?tNO;17M4UF=%q
6
set_parent
5
0
set_parent
5
6
allocate
/RXQV%#ZRs/Wc
5
set_parent
5
0
allocate
B&4(Hun?9!IW+
2
allocate
l)x,g
2
allocate
$NQM+c=
4
set_parent
2
0
set_parent
2
6
allocate
BvgK
4
ref_node
4
allocate
pt9c~wVW
1
allocate
d92r-
3
allocate
,X}YuD#5
7
ref_node
7
allocate
uzXD,e%
1
ref_node
11
allocate
IL`\K*pu]%C6{X/U
1
allocate
~/A!c4q/Y*1iDE~q
9
allocate
eZ15NR7
5
set_parent
11
10
remove
13
remove
5
remove
9
allocate
:{50Y%T\S+l
8
ref_node
8
allocate
HiW[[}x>uY
0
allocate
cgdCHYt)?;MXgzHc
0
allocate
1v}1
10
allocate
51BD9n{@U
11
remove
15
set_parent
94204233577824
20
allocate
taWg!:fV-nC
0
set_parent
3
11
remove
6
set_parent
14
15
allocate
Al_k&tz8Fd
0
allocate
%D57[?YwR)Rf
13
set_parent
8
14
ref_node
15
ref_node
15
set_parent
14
13
allocate
:y2JJch
13
set_parent
15
5
allocate
8?GJWy#\
5
peek_nodes
set_parent
13
15
allocate
=v^c[o<v
14
ref_node
15
allocate
8?C&r}K##fxN]
0
set_parent
1
0
remove
12
allocate
a-h`
5
allocate
"vuw_%R0
14
set_parent
11
10
remove
10
set_parent
3
5
allocate
Qo6gnYm.m8
15
allocate
6z\Z`%qPU]
13
remove
3
allocate
N9%V5b4gt3pW8
14
allocate
=&]16hZ\D=k=-.rL
8
allocate
LzC7xp^IjCnHH.
0
allocate
A^m}KBQkSc_!@Qz
15
set_parent
15
9
remove
2
set_parent
12
14
remove
7
set_parent
4
8
remove
15
allocate
CY0n{I!S?B%'1Y_(
0
set_parent
5
1
remove
9
set_parent
20
140732017153072
set_parent
10
1
remove
10
set_parent
5
0
allocate
F&U8\9R<o,.
1
set_parent
6
3026031
set_parent
3
5
allocate
z}+AbW-Wq>r4
3
ref_node
5
ref_node
3
allocate
b)9&.PBH}
3
remove
2
allocate
nlPl\
3
allocate
8A+?TzZK?
5
allocate
t#lr{
0
allocate
/la-e;u]
5
allocate
OzPXG;o
0
allocate
.?T!f`9c3Rp?
1
remove
4
ref_node
7
set_parent
13
5
allocate
8~NQ7e
3
set_parent
13
11
remove
7
ref_node
12
allocate
+zcu
1
remove
12
allocate
*fh1%Y}b1N#QuS
19
allocate
!FJDRL~GEq@aR.B
3
allocate
EuV5FKj11]fj
5
set_parent
1
5
allocate
Mx1cA>aN
0
set_parent
11
9
remove
15
allocate
JqsjVscVvJK#b
11
allocate
6Ou*O/nm$z_kyD
13
allocate
l'!j96I?"`
8
remove
6
remove
11
allocate
R4H.aL
9
ref_node
3
allocate
Zn_DNv
2
allocate
:OO?rrkZ}_4O
15
remove
8
allocate
@N`<B\]K{cUv#x9
19
allocate
nZmT
5
allocate
_{avH{QyMaF;2:
3
allocate
hPW?
7
allocate
ysTr'3L
1
allocate
"x`!LvraNmG&+n
14
ref_node
8
allocate
k*t~PvZZZ_#
4
ref_node
16
peek_nodes
allocate
6b>.Q
7
remove
4
ref_node
5
allocate
Sy'dIB4K
6
allocate
37~+*Ch
18
remove
9
set_parent
6
14
allocate
rXkv+(IB1r%<Q%
14
allocate
dw[0M
3
allocate
g8D][o$->A*x
9
allocate
)*>@"
8
remove
8
remove
13
allocate
dcuv1
2
remove
4
allocate
s}Bsc5C36r
1
allocate
K'GQqbViL4;7lvg]
10
allocate
tcw>G%jPmTekU
16
allocate
PC_3*TQ
17
remove
17
set_parent
12
0
allocate
Z9*h9+
3
ref_node
7
allocate
F5"SfOV{~deOT0
9
set_parent
11
9
allocate
:8$z,2#
4
remove
15
ref_node
9
allocate
2r;&n2&uq
7
allocate
tq2_v]\nxn
9
set_parent
14
15
allocate
F55aWu*
0
set_parent
15
12
remove
3
remove
4
remove
18
allocate
>&B7Ak4'A4eiI
15
allocate
\\s]5h,
7
quit
