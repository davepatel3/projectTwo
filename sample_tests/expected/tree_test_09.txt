allocate
1
allocate
2
allocate
3
remove
Invalid handle!
allocate
4
set_parent
remove
ref_node
c_:Z
parent: 1
children: 
allocate
3
allocate
2
allocate
5
allocate
6
allocate
7
ref_node
/Wx6L
parent: 4
children: 
remove
allocate
6
allocate
4
set_parent
ref_node
;iJ</AN}M,
parent: 0
children: 
ref_node
;iJ</AN}M,
parent: 0
children: 
allocate
7
allocate
2
set_parent
allocate
5
set_parent
Invalid handle!
remove
remove
peek_nodes
0: 
1: [recycled]
2: [recycled]
3: [recycled]
4: [recycled]
5: [recycled]
6: [recycled]
7: [recycled]
allocate
3
allocate
1
remove
allocate
7
set_parent
remove
remove
allocate
6
allocate
5
set_parent
set_parent
allocate
2
set_parent
allocate
4
allocate
3
allocate
1
set_parent
set_parent
allocate
7
ref_node
B&4(Hun?9!IW+
parent: 2
children: 1 7 
allocate
8
allocate
9
allocate
10
ref_node
BvgK
parent: 4
children: 10 
allocate
11
ref_node
uzXD,e%
parent: 1
children: 
allocate
12
allocate
13
allocate
14
set_parent
remove
remove
remove
allocate
13
ref_node
pt9c~wVW
parent: 1
children: 13 
allocate
14
allocate
5
allocate
9
allocate
15
remove
set_parent
Invalid handle!
allocate
15
set_parent
remove
set_parent
allocate
13
allocate
8
set_parent
ref_node
taWg!:fV-nC
parent: 0
children: 14 
ref_node
taWg!:fV-nC
parent: 0
children: 14 
set_parent
allocate
12
set_parent
allocate
1
peek_nodes
0: 
1: 8?GJWy#\
2: [recycled]
3: [recycled]
4: [recycled]
5: cgdCHYt)?;MXgzHc
6: [recycled]
7: [recycled]
8: %D57[?YwR)Rf
9: [recycled]
10: [recycled]
11: [recycled]
12: :y2JJch
13: Al_k&tz8Fd
14: HiW[[}x>uY
15: taWg!:fV-nC
set_parent
allocate
3
ref_node
taWg!:fV-nC
parent: 5
children: 13 
allocate
11
set_parent
remove
allocate
9
allocate
10
set_parent
remove
set_parent
allocate
7
allocate
4
remove
allocate
2
allocate
6
allocate
12
allocate
11
set_parent
remove
set_parent
remove
set_parent
remove
allocate
10
set_parent
remove
set_parent
Invalid handle!
set_parent
remove
set_parent
allocate
3
set_parent
Invalid handle!
set_parent
allocate
2
ref_node
cgdCHYt)?;MXgzHc
parent: 0
children: 3 
ref_node
F&U8\9R<o,.
parent: 5
children: 2 
allocate
7
remove
allocate
6
allocate
4
allocate
8
allocate
12
allocate
14
allocate
13
remove
ref_node
b)9&.PBH}
parent: 3
children: 
set_parent
allocate
11
set_parent
remove
ref_node
/la-e;u]
parent: 5
children: 
allocate
15
remove
allocate
Invalid handle!
allocate
9
allocate
10
set_parent
allocate
2
set_parent
remove
allocate
4
allocate
7
allocate
12
remove
remove
allocate
15
ref_node
F&U8\9R<o,.
parent: 5
children: 9 
allocate
6
allocate
7
remove
allocate
Invalid handle!
allocate
13
allocate
4
allocate
11
allocate
12
allocate
8
ref_node
"x`!LvraNmG&+n
parent: 14
children: 
allocate
16
ref_node
k*t~PvZZZ_#
parent: 4
children: 
peek_nodes
0: 
1: 8?GJWy#\
2: Mx1cA>aN
3: F&U8\9R<o,.
4: _{avH{QyMaF;2:
5: cgdCHYt)?;MXgzHc
6: Zn_DNv
7: :OO?rrkZ}_4O
8: "x`!LvraNmG&+n
9: !FJDRL~GEq@aR.B
10: EuV5FKj11]fj
11: hPW?
12: ysTr'3L
13: nZmT
14: OzPXG;o
15: R4H.aL
16: k*t~PvZZZ_#
allocate
17
remove
ref_node
cgdCHYt)?;MXgzHc
parent: 0
children: 3 10 1 13 
allocate
16
allocate
Invalid handle!
remove
set_parent
allocate
4
allocate
11
allocate
Node is recycled!
allocate
17
remove
remove
allocate
7
remove
allocate
15
allocate
9
allocate
17
allocate
8
remove
set_parent
allocate
13
ref_node
dcuv1
parent: 2
children: 
allocate
4
set_parent
allocate
8
remove
ref_node
K'GQqbViL4;7lvg]
parent: 10
children: 4 11 
allocate
17
allocate
15
set_parent
allocate
18
set_parent
remove
remove
remove
allocate
13
allocate
3
quit
